types/metadataV14.go

```diff
@@ -160,12 +160,12 @@ func (m *MetadataV14) FindError(moduleIndex U8, errorIndex U8) (*MetadataError,
 				}
 
 				for _, variant := range errType.Def.Variant.Variants {
-					if variant.Index == errorIndex {
+					if variant.Index == errorIndex[0] {
```

Comment:
If the [0] is the error index in the module/pallet, the what are the other 3 positions for? Looking at the metadata it seems that the index is a uint, so are the other bytes empty? or do we need to convert that byte array to number?

Reply:
I was not able to find more information regarding those other 3 indexes, that's why locally I tested it with that and the first index seems to be the one that works. And tbh this is still confusing me. Probably we can take a look at that together to figure how it's supposed to be implemented here.

Reply:
As discussed, let's keep this for the time being, since it's also similar to how it's done in `polkadot-js`.